server/services/mapping_service.py

```diff
@@ -138,8 +138,24 @@ def generate_gpx(project_id: int, task_ids_str: str, timestamp=None):
         return xml_gpx
 
     @staticmethod
-    def generate_osm_xml():
-        # TODO remove or productionise
-        tree = ET.parse('osm_sample.xml')
-        root = tree.getroot()
-        return ET.tostring(root, encoding='utf8')
+    def generate_osm_xml(project_id: int, task_ids_str: str) -> str:
+        """ Generate xml response suitable for loading into JOSM.  A sample output file is in 
+            /server/helpers/testfiles/osm-sample.xml """
+        root = ET.Element('osm', attrib=dict(version='0.6', upload='no', creator='HOT Tasking Manager'))
```

Comment:
maybe highlight the upload = no attaribut, as that't important to control JOSM not uploading the data to OSM